apps/admin-gui/src/app/shared/components/dialogs/edit-email-footer-dialog/edit-email-footer-dialog.component.html

```diff
@@ -2,11 +2,24 @@
   <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.TITLE' | translate}}</h1>
   <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
   <div *ngIf="!loading" class="dialog-container" mat-dialog-content>
-    <div>{{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.LABEL' | translate}}:</div>
-
-    <mat-form-field>
-      <textarea [(ngModel)]="mailFooter" [cdkTextareaAutosize]="true" matInput></textarea>
-    </mat-form-field>
+    <perun-web-apps-alert alert_type="info" *ngIf="plainEdithAuth && !htmlEditAuth">
+      {{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.FORMAT_HTML_NOT_AUTHORIZED' | translate}}
+    </perun-web-apps-alert>
+    <mat-tab-group color="primary">
+      <mat-tab *ngFor="let format of formats" [disabled]="format === 'html' && !htmlEditAuth">
+        <ng-template mat-tab-label>
+          <span [ngClass]="{'disabled-label': format === 'html' && !htmlEditAuth}">
```

Comment:
![image](https://user-images.githubusercontent.com/70371638/185169863-1e25dc9d-6fda-4d6d-aaba-23152b3e6619.png)
You need to add the `.disabled-label` class also to the edit-email-footer-dialog.component.scss file (it's not the global class, so this class is not applied to this span now):
```
.disabled-label {
  color: rgba(0, 0, 0, 0.38) !important;
}
```
![image](https://user-images.githubusercontent.com/70371638/185170578-4bf86ba7-93ba-410f-9d6d-ea6a98bc5fbf.png)